perl/share/Rscripts/get_rg_cns.R

```diff
@@ -361,7 +361,7 @@ while (i <= length(seg_chr)) {
 
      #sb43-count number of reads in  subset file, if mpty skip record
     cmd = paste0(
-        "samtools view -c", tmp_bam, ".subset"
+        "samtools view -c ", tmp_bam, ".subset"
```

Comment:
Does the command execution allow pipes?  I'm just toying with the idea that instead of counting all of the reads in a region (parsing the whole file) you could just ensure at least 1 exists with:

`samtools view in.bam.subset | head -n 1 | wc -l`

Sorry only just thought of it.  Otherwise it looks good